private/bufpkg/bufcheck/buflint/testdata/validate_rules_types_match_errors/buf.yaml

```diff
@@ -0,0 +1,7 @@
+version: v1
+lint:
+  use:
+    - VALIDATE_CONSTRAINTS_CHECK
+  ignore:
+    - buf
+
```

Comment:
Nit: extra newline

Reply:
resolved in [a3c77f8](https://github.com/bufbuild/buf/pull/2377/commits/a3c77f8869438dc13611808c7e3d10bcdbf8dc1f)